attachments.md

```diff
@@ -35,6 +35,6 @@ small_cover_land.jpg | Landscape          | 120
 
 Font files MAY be added to a Matroska file as Attachments so that the font file may be used to display an associated subtitle track. This allows the presentation of a Matroska file to be consistent in various environments where the needed fonts might not be available on the local system.
 
-To associate a font file with a subtitle track, the font file MUST be stored as an Attachment within the same Segment. The `TrackEntry Element` of the associated subtitle track SHOULD store an `AttachmentLink Element` that contains the value of the `FileUID Element` of the `AttachedFile Element` that stores the font file.
+To associate a font file with a subtitle track, the font file MUST be stored as an Attachment within the same Segment.
 
-For subtitle formats that reference a required font by name, such as SubStation Alpha (SSA/ASS), the `AttachmentLink Element` is RECOMMENDED. If no `AttachmentLink Element` is used then the Matroska Reader SHOULD use the font(s) stored in Attachments that match the names of those needed by the subtitle encoding. If the subtitle encoding requires fonts that are not specified by an `AttachmentLink Element` or use names that do not match the `FileName` of any `AttachmentFile Elements` then the Matroska Reader SHOULD attempt to find a system font to use with the subtitle track.
+For subtitle formats that reference a required font by name, such as SubStation Alpha (SSA/ASS), the Matroska Reader SHOULD use the font(s) stored in Attachments that match the names of those needed by the subtitle encoding. If the subtitle encoding requires fonts that use names that do not match the `FileName` of any `AttachmentFile Elements` then the Matroska Reader SHOULD attempt to find a system font to use with the subtitle track.
```

Comment:
I don't like this. There are tons of fonts whose name doesn't match their file name. I'd prefer our recommendation to state that attached fonts SHOULD be registered with the font system temporarily during playback and that name matching SHOULD be done afterwards with preference given to the fonts attached fonts (if the font system allows such fine-grained control).

Reply:
IMO we shouldn't have to say how to use a font. This seems to be system specific. It's also possible to use a font without registering it in the system (in VLC we don't use the system font rendering system).
But font names and file names are clearly not the same thing. We should only say that the font SHOULD be used if its name matches the font needed by the subtitle format.

It's also format specific, some formats may have different font names in decreasing preference order, some may require a general format, etc
The fallback may also be defined/overriden by the user (serif, sans-serif, monotype, bold, thin, etc).